lib/chrome.js

```diff
@@ -75,7 +74,7 @@ module.exports = class ChromeWrapper {
       '--headless',
       '--single-process',
       '--remote-debugging-port=9222',
-      `--window-size=${width},${height}`,
+      `--window-size=800,600`,
```

Comment:
Can you tell me more about this change? Just favoring the window size in individual tests instead?

Reply:
The Zen.config is not defined in lambda so I am just removing that part the code. Also, now that tests can define it themselves this code is basically useless.